samples/word/99-preview-apis/manage-comments.yaml

```
@@ -0,0 +1,335 @@
+id: word-manage-comments
+name: Manage comments
+description: 'This sample shows how to perform operations on comments (including insert, reply, get, edit, resolve, and delete) and use comment events.'
+host: WORD
+api_set:
+    WordApi: '1.8'
+script:
+    content: |
+        $("#register-event-handlers").on("click", () => tryCatch(registerEventHandlers));
+        $("#insert").on("click", () => tryCatch(insertComment));
+        $("#edit").on("click", () => tryCatch(editFirstCommentInSelection));
+        $("#reply").on("click", () => tryCatch(replyToFirstActiveCommentInSelection));
+        $("#resolve").on("click", () => tryCatch(toggleResolvedStatusOfFirstCommentInSelection));
+        $("#range").on("click", () => tryCatch(getFirstCommentRange));
+        $("#get-comments-in-selection").on("click", () => tryCatch(getCommentsInSelection));
+        $("#delete").on("click", () => tryCatch(deleteFirstCommentInSelection));
+        $("#get-comments").on("click", () => tryCatch(getComments));
+        $("#deregister-event-handlers").on("click", () => tryCatch(deregisterEventHandlers));
+        $("#setup").on("click", () => tryCatch(setup));
+
+        let eventContexts = [];
+
+        async function registerEventHandlers() {
+          // Registers event handlers.
+          await Word.run(async (context) => {
+            const body = context.document.body;
+            body.track();
+            await context.sync();
+
+            eventContexts[0] = body.onCommentAdded.add(onEventHandler);
+            eventContexts[1] = body.onCommentChanged.add(onChangedHandler);
+            eventContexts[2] = body.onCommentDeleted.add(onEventHandler);
+            eventContexts[3] = body.onCommentDeselected.add(onEventHandler);
+            eventContexts[4] = body.onCommentSelected.add(onEventHandler);
+            await context.sync();
+
+            console.log("Event handlers registered.");
+          });
+        }
+
+        async function insertComment() {
+          // Sets a comment on the selected content.
+          await Word.run(async (context) => {
+            const text = $("#comment-text")
+              .val()
+              .toString();
+            const comment = context.document.getSelection().insertComment(text);
+
+            // Load object for display in Script Lab console.
```

Here and elsewhere, consider avoiding mentioning Script Lab, since these samples could appear elsewhere.
```suggestion
            // Load object to log in the console.
```